article/models.py

```diff
@@ -6,13 +6,13 @@
 class Article(models.Model):
     id = models.UUIDField(default=uuid.uuid4,  unique=True, primary_key=True, editable=False)
     user = models.ForeignKey(User,on_delete=models.SET_NULL, null=True, blank=True)
-    title = models.TextField(max_length=500, default="untitled")
+    title = models.CharField(max_length=500, default="untitled")
```

Comment:
Any reason we had to change this to `CharField`? I believe there isn't a difference

Reply:
There will never be so long a title anyone will ever put ! Some changed the code and i dont know who pushed the code without discussion!

Reply:
I see what you mean. Perfect sense!

Reply:
Yep! Let me know if everything is good to go or not?